vuepress/docs/next/tutorials/samples/customer-portal.md

```diff
@@ -0,0 +1,166 @@
+---
+sidebarDepth: 2
+---
+
+# Entando Customer Portal 
+## Overview
+The Entando Customer Portal enables an organization to quickly provide a modern, self-service customer-facing application for management of subscriptions. It includes a lightweight integration to JIRA Service Management for access to service tickets and a Role Based Access Control (RBAC) design for granting users varying levels of access.
+
+Key Features:
+
+* Customizable service ticket and tracking system with Jira Service Management
+* Role-based access control (RBAC) with Entando Identity Management System
+* Integrated user, customer, project, and subscription management 
+
+This tutorial covers: 
+1. [Installation](#installation)
+2. [Configuration](#configuration)
+4. [Managing the Customer Portal](#managing-the-customer-portal)
+5. [Using the Customer Portal](#using-the-customer-portal)
+6. [Application Details](#application-details)
+7. [Resources](#resources)
+
+## Installation
+### Prerequisites
+* A working instance of Entando running on Kubernetes. See [Getting Started](../../docs/getting-started/) for more information or [install Entando on any Kubernetes provider](../#operations). 
+* Use the Entando CLI command `ent check-env develop` to verify all dependencies
+* Docker installed and running
+
+### Automatic Install with App Builder
+Install the Customer Portal in a few simple steps by integrating the Entando Hub into your App Builder. In the Hub, you can update or roll back versions of a bundle at any time.
+1. From your App Builder, go to the Repository from the left navigation bar. Enter Entando Hub in the Registry field at the top of the page. You will now have access to the Hub.
+
+2. `Install` the two bundles for the Customer Portal from the Catalog. Once complete, the Portal is ready for use. 
+
+To go directly to your Portal, navigate to `Pages Management`, use the `Actions` for the Customer Portal and select `View Published Page`.
+
+![Entando Hub](./images/custportal-entando-hub.png)
+
+### Manual Install 
+1. To install the Customer Portal, run the following commands in the listed order. Edit the `-n entando` option for each command to match your namespace or project.
+
+``` bash
+ent bundler from-git -r https://github.com/nshaw/customerportal-wip-bundle.git -d | ent kubectl apply -n entando -f - 
+```
+
+``` bash
+ent bundler from-git -r https://github.com/nshaw/customerportal-content-bundle.git -d | ent kubectl apply -n entando -f - 
+```
+2. Log into your App Builder 
+
+3. Select Repository from the left menu. Two Customer Portal bundles will be visible in the repository. `Install` both bundles, in any version desired. 
+
+4. To navigate to your Portal: \
+from the left menu →  `Page` → `Mangement` \
+find `Customer Portal` under `Page Tree` list \
+from the `Actions` pull-down menu →  `View Published Page`
+
```

Comment:
I'm not sure where it should live but we still need the config steps for 1) enabling the mail mapper in keycloak per https://github.com/entando-samples/customer-portal/tree/main/application "The built-in mapper for email must be enabled on the server client so that user accounts can be retrieved from Jira and tickets created used that account information.", and 2) using the cp_admin_config.page to configure the Customer Portal to connect to JIRA Service Management.

Reply:
As a TODO we still also need a way to configure the Customer Portal to let it work with a specific Jira Service Management instance. There are a couple tickets open to finish that work.